### DISCUSSION TITLE
Monospaced font

### DISCUSSION THREAD

**Discussion by jjangga0214** [orioncactus/pretendard#44] [Q&A] [answered]
안녕하세요?

좋은 폰트를 오픈소스로 공유하여 감사드립니다.

모든 글자의 크기(너비)가 같은 Monospaced 형식의 폰트는 계획에 없으실까요?

감사합니다.

**Comment by orioncactus** [comment]
안녕하세요! Pretendard를 아껴주시고 관심을 가져주셔서 감사드립니다.

고정폭에 맞춘다면 다른 고정폭 글꼴과 같이 형태를 고정폭에 맞게 다듬어야 할 듯하며, 아직까지는 해당 작업의 필요성을 못 느껴 고정폭 Pretendard는 고려하고 있지 않습니다. Pretendard는 다국어 표시와 관련한 본문 글꼴의 병목 최소화에 방향을 두고 있습니다.

아래 이슈와 이 이슈가 서로 동일해 아래 이슈는 삭제하고, 해당 이슈는 질문인 듯해 Discussions으로 옮기겠습니다. 추가적으로 궁금하신 부분이 있다면 Discussions에서 언제든지 알려주세요! ᏊˊꈊˋᏊ

감사합니다!

**Comment by jjangga0214** [comment, marked as answer]
@orioncactus 혹시 프로그래밍 용도로 사용할 수도 있을 것 같아서 여쭤보았습니다. 
만약 고정폭 글꼴로 나온다면 프로그래밍 용도로 사용하기에 적합성이 있을 것으로 생각하시나요?

**Reply by orioncactus** [reply]
고정폭 글꼴과 본문용 글꼴은 서로 모양에 큰 차이가 있어 지금 Pretendard에서 폭을 수정하고 Powerline용 글리프를 더해 배포하는 것으로는 고정폭 글꼴 수요자에게 적합해보이진 않습니다. Pretendard를 적합한 코드 에디터용 고정폭 글꼴로 만들기 위해서는 Pretendard에 포함된 글자 하나하나를 수정해야 하는데, 사실 지금까지 만든 시간 이상으로 힘을 들여야 할 듯하고, 또 제가 코드 에디터를 사용하면서 글꼴에 대해 큰 불편함이 없어서, 고정폭 Pretendard는 아직까지는 유의미하게 고려하고 있지 않습니다. 혹시 코드 에디터를 목적으로 사용할 수 있는 다른 한글 글꼴을 찾으신다면, 대안으로 [나눔고딕 D2Coding](https://github.com/naver/d2codingfont)을 추천드립니다.

**Reply by gintering1000** [reply]
요청자께서는 Pretendard만 설치하면 코드, GUI, 문서 등 고딕에 관한 제작자를 위한 모든 준비가 끝나길 원하시는 것 같습니다. 저 또한 이렇게 완벽한 폰트를 근래에 본적이 없어 질문자와 같은 생각을 하지 않은것은 아닙니다. ( 저 또한 소망합니다 ) 지금은 말씀하신것처럼 목표가 "다국어 표시와 관련한 본문 글꼴의 병목 최소화"와 같이 세부적이지만 추후 더욱 큰프로젝트로 확장한다면, 네이버 처럼 패키지를 분리하지 않고, "코딩+GUI+문서 등의 통합 고딕"의 목표로 출시를 했으면하는 바램입니다.
